### PR TITLE
completed assignment 1

### PR DESCRIPTION
**Pull request by thakaredipali**
1. Write a program to calculate the simple interest 
First-line has the comma-separated values of the Principal, rate and time (in years) respective
*constraints: Round simple interest float value to 2 decimal places
2. Write a program to calculate the area of the circle, First line has a value of the radius of the circle
constraint
1. Use const PI from the package math package
2. Use the Pow function from the math package
3. Round area float value to 2 decimal places

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import (
+	"fmt"
+	"go-lang-assignments/utils"
+	"strconv"
+	"strings"
+)
+
+func main(){
+
+   /// Calculate simple interest
```
Comment (Deepak1124) [inline main.go]: 2 backslashes are enough to mark it as comment.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	var input string
+	fmt.Println("Enter Principal, Rate, and Time:")
+	 _, err:= fmt.Scan(&input)
+
+	if err != nil{
+     fmt.Println("Invalid Input")
```
Comment (Deepak1124) [inline main.go]: Pls take care of indentation, you can use tab.

File: main.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	 return 
+	}
+	values := strings.Split(input, ",")
```
Comment (Deepak1124) [inline main.go:21]: you could have used "fmt.Scanln", then you don't need to split the input.

**Review by Deepak1124 — CHANGES_REQUESTED** [review]

File: main.go (hunk 3 above)
Comment (thakaredipali) [reply main.go:21]: In Question there is mention "comma-separated values of the Principal, rate and time". that's why I'm using split.

**Pull request merged**